.claude/skills/bear-review/SKILL.md

```
@@ -15,6 +15,124 @@ description: BEAR.SundayプロジェクトのPHPコード品質を評価する
+echo "CyclomaticComplexity:   $(grep -c 'CyclomaticComplexity' phpmd_output.txt) 件"
+echo "NPathComplexity:        $(grep -c 'NPathComplexity' phpmd_output.txt) 件"
+echo "ExcessiveMethodLength:  $(grep -c 'ExcessiveMethodLength' phpmd_output.txt) 件"
+echo "ExcessiveClassLength:   $(grep -c 'ExcessiveClassLength' phpmd_output.txt) 件"
+echo "TooManyFields:          $(grep -c 'TooManyFields' phpmd_output.txt) 件"
+```
+
+#### 統計レポート例
+
+実行結果の例：
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)